App.tsx

```diff
@@ -84,72 +86,34 @@ export default function App() {
         if (enabled) {
           console.log("Authorization status:", authStatus);
 
-          // Get the token
-          const fcmToken = await messaging().getToken();
-          if (fcmToken) {
-            console.log("Your Firebase Cloud Messaging token is:", fcmToken);
-
-            // Subscribe to the "all" topic
-            await messaging()
-              .subscribeToTopic("All")
-              .then(() => console.log("Subscribed to 'All' topic!"));
-
-            // Register the token for the "all" topic with the backend
-            try {
-              const allRegister: RegisterUserDTO = {
-                Token: fcmToken,
-                Topic: "All", // This should match the topic defined in your backend
-              };
-              const allResponse = await API.firebase.registerFirebase(
-                allRegister
-              );
-              console.log(
-                "Firebase 'All' registration response: ",
-                allResponse
-              );
-            } catch (error) {
-              console.error("Error during Firebase All' registration:", error);
-            }
-
-            // Subscribe to the "volunteer" topic
-            await messaging()
-              .subscribeToTopic("arkad")
-              .then(() => console.log("Subscribed to 'arkad' topic!"));
-
-            // Register the token for the "volunteer" topic with the backend
-            try {
-              const volunteerRegister: RegisterUserDTO = {
-                Token: fcmToken,
-                Topic: "arkad", // This should match the topic defined in your backend
-              };
-              const volunteerResponse = await API.firebase.registerFirebase(
-                volunteerRegister
-              );
-              console.log(
-                "Firebase 'arkad' registration response: ",
-                volunteerResponse
-              );
-            } catch (error) {
-              console.error(
-                "Error during Firebase 'arkad' registration:",
-                error
-              );
-            }
-          } else {
-            console.log("Failed to get FCM token");
-          }
-        }
-      }
 
-      const unsubscribe = messaging().onMessage(async (remoteMessage: any) => {
-        Alert.alert(
-          "A new FCM message arrived!",
-          JSON.stringify(remoteMessage)
+    // Get the token
+    const { data: expoPushToken } = await Notifications.getExpoPushTokenAsync();
+
+    if (expoPushToken) {
+      console.log("Your Expo push token is:", expoPushToken);
+
+      // Register the token with the backend
+      try {
+        const allRegister: RegisterUserDTO = {
+          Token: expoPushToken,
+          userId: (await API.users.getMe()).id.toString()
```

Comment:
Istället för att göra såhär, vill jag istället att ni använder er av faktumet att `postAuth` brukas. Så backenden vet redan vilken användare som registrerar sig bara genom att kolla vilken användare skickar upp DTOn. Att be frontend identifiera sig själv är en **dålig** idé utifrån flera perspektiv, en person hade kunnat skicka upp att dem var vilken användare som helst och det känns inte jättebra ur ett säkerhetsperspektiv...